scripts/test.sh

```diff
@@ -22,6 +22,7 @@ run () {
     yarn mocha $(find desktop/apps/*/test -name '*.coffee')
     yarn mocha $(find desktop/apps/*/test -name '*.js')
     yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee')
+    yarn mocha $(find desktop/apps/**/__tests__ -name '*.js')
```

Comment:
Following in line with patterns in `reaction-force` and past experience showing that co-located tests **vastly** increases test coverage, I've moved our `auctions2` tests out of a root test folder to support this new structure. In another PR i'll clean everything up and properly break things out but for now just know that this has been updated.